package/autoyast2.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Fri Jun  9 09:08:50 CEST 2017 - schubi@suse.de
+
+- Report shrinked partitions if the requirements have not met
```

Comment:
np: I would say something like `if the requirements have not been met` or just `if there is not enough space`.